### PR TITLE
Modify the .toml files to add new versions

### PR DESCRIPTION
**Pull request by pablo-herranz**

### CODE REVIEW

File: _review/opensuse-microos.toml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 Instance = "https://openqa.opensuse.org"                      # openQA instance to query
 RabbitMQ = "amqps://opensuse:opensuse@rabbit.opensuse.org"    # RabbitMQ instance to query
 RabbitMQTopic = "opensuse.openqa.job.done"                    # RabbitMQ topic to query
-HideStatus = [ "scheduled","passed","assigned" ]              # Hide scheduled, passed and assigned jobs
+HideStatus = [ "scheduled", "passed", "softfailed", "running", "reviewed" ] # Hide scheduled, passed and assigned jobs
```
Comment (jlausuch) [inline _review/opensuse-microos.toml:6]: maybe `soft-failed` can be removed? what is `reviewed` ?

Comment (pablo-herranz) [reply _review/opensuse-microos.toml:6]: I think reviewed are those which are failed but already have a poo ticket.
You mean to display all the `softfailed`? :exploding_head:

Comment (m-dati) [reply _review/opensuse-microos.toml:6]: The comment, what it is hiding, should be updated too, in all the _toml_ where present, but without repeating the list of cases, only: `hide jobs in a specific status`

Comment (jlausuch) [reply _review/opensuse-microos.toml:6]: ah ok sorry, I understood it the otherway around, soft-fail dones't need to be displayed, yes, same as jobs with tickets.

**Pull request closed without merging**